src/app/components/hmcts-global-header/hmcts-global-header.component.ts

```diff
@@ -27,9 +27,19 @@ export class HmctsGlobalHeaderComponent {
   public showItems: boolean;
   public userValue = true;
   public tab;
+  public leftItems: NavItemsModel[];
+  public rightItems: NavItemsModel[];
 
   constructor(
     public nocStore: Store<fromNocStore.State>) { }
+  ngOnChanges(changes: SimpleChanges): void {
+    this.splitNavItems();
+  }
+
+  public splitNavItems() {
+    this.rightItems = this.items.filter(item => item.align && item.align === 'right');
```

Comment:
`splitNavItems()` runs when any of the `@Input()` properties change. Can we make  `splitNavItems()` run only when the interested property changes are made?

`if(changes['items']) { // rest of the code}
`

Reply:
Good point. Fixed it now